webpack/components/extensions/RegistrationCommands/fields/LifeCycleEnvironment.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  FormSelectOption, FormSelect,
+} from '@patternfly/react-core';
+
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { noop } from 'foremanReact/common/helpers';
+
+const LifeCycleEnvironment = ({
+  pluginValues, onChange, isLoading,
+  hostGroupEnvironment, lifeCycleEnvironments,
+}) => (
+  <FormGroup
+    label={__('Lifecycle enviroment')}
+    fieldId="reg_katello_lce"
+    helperText={hostGroupEnvironment && sprintf('From host group: %s', hostGroupEnvironment)}
```

Comment:
I think something like Host Group should be capitalized? Not 100% sure on that. Where's our style guide?? :)

Reply:
Me neither, according to this [comment](https://github.com/theforeman/foreman/pull/8419#issuecomment-823401017) it should be `host group`, since it's not name or a label for input.